doc/help.rst

```diff
@@ -6,9 +6,10 @@ You can use the **Help** pane to find, render and display rich documentation for
 
 |
 
-.. raw:: html
-
-	<p align= "center"><iframe src="https://giphy.com/embed/H1quFSJ3LSwmVwvIhU" width="600" height="375" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/H1quFSJ3LSwmVwvIhU">via GIPHY</a></p></p>
+.. image:: images/help/help-execution.gif
```

Comment:
@juanis2112, I think this screenshot is pretty good but it has one problem: you didn't use Spyder 4.1 to film it because it's not showing the message "Retrieving documentation" that we show now.